includes/Core/Permissions/Permissions.php

```diff
@@ -260,4 +260,24 @@ private function grant_additional_caps( array $allcaps ) {
 
 		return $allcaps;
 	}
+
+	/**
+	 * Get all capabilities used in Google Site Kit.
```

Comment:
Minor nitpick, per our documentation standards function/method docs should start with 3P singular verbs.
```suggestion
	 * Gets all capabilities used in Google Site Kit.
```